x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/handlers.ts

```diff
@@ -13,6 +13,35 @@ import {
 import { EndpointAppContext } from '../../types';
 import { exceptionItemToTrustedAppItem, newTrustedAppItemToExceptionItem } from './utils';
 import { ENDPOINT_TRUSTED_APPS_LIST_ID } from '../../../../../lists/common/constants';
+import { DeleteTrustedAppsRequestParams } from './types';
+
+export const getTrustedAppsDeleteRouteHandler = (
+  endpointAppContext: EndpointAppContext
+): RequestHandler<DeleteTrustedAppsRequestParams, undefined, undefined> => {
+  const logger = endpointAppContext.logFactory.get('trusted_apps');
+
+  return async (context, req, res) => {
+    const exceptionsListService = endpointAppContext.service.getExceptionsList();
+
+    try {
+      const { id } = req.params;
+      const response = await exceptionsListService.deleteExceptionListItem({
+        id,
+        itemId: undefined,
```

Comment:
this is something Exception list specific?

Reply:
Yes, its specific to the implementation of Exceptions Lists (List?). `id` is the Saved Object ID (if known), while `itemId` is a generated value that you can store in the saved object list of attributes.